### ISSUE TITLE
Only return extended package dict, if it has been requested explicitly

### ISSUE THREAD

**Issue by metaodi** [transparenzportalhamburg/ckanext-fulltext#3]
I propose to not return the extended package dict with the fulltext, unless it has been requested explicitly.

I would image the `package_show_minimal` to have a check like this:

``` python
if check_logged_in(context) and context.get('return_extended', False):
```

Like this all CKAN internal calls to `package_show` get exactly what they need, and all explicit calls from outside (e.g. from an extension) have an easy possibility to get the full package dict.

What do you think? I can provide a PR if you like.

**Comment by hotz** [comment]
Yes, we can think about such a facility (e.g. also to have a configuration parameter in development.ini).
The reason for having it not (or for defining  context.get('return_extended', True) ) is that the plugins which do not know about this fulltext extension should deal with the full_text_search property like they do with others. E.g. now the CKAN-dashboard (Web-GUI) just presents the field in the detail view. 
With a parameter return_extended which defaults to False this Gui would not present the field.

Perhaps default to True would solve it.... New plugins could use the parameter for others it would be transparent.

Or, have a configuration parameter...
Not sure, will discuss with further developers...
